### PR TITLE
Implement New toolkit version csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List -** 

**1.meetings-share-to-stage-signing/csharp
2.meetings-stage-view/csharp
3.meetings-token-app/csharp
4.msgext-search-sso-config/csharp**

### CODE REVIEW

File: samples/meetings-token-app/csharp/TeamsApp/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
Comment (Alive-Fish) [inline samples/meetings-token-app/csharp/TeamsApp/launchSettings.json:6]: The `launchUrl` seems wrong. Please use `${{TEAMS_APP_ID}}` replacing `%TEAMSAPPID%`, also, check other variables and `launchUrl`.

**Review by Alive-Fish — APPROVED** [review]
Only find the following obvious issue. Please check it.

File: samples/meetings-token-app/csharp/TeamsApp/launchSettings.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/meetings-token-app/csharp/TeamsApp/launchSettings.json:6]: Fixed

**Review by Alive-Fish — APPROVED** [review]

**Review by Mohammed-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11328308124

**Review by Jegadeesh-MSFT — APPROVED** [review]
Looks good !

**Pull request merged**